app/src/main/java/eu/kanade/presentation/library/components/CommonMangaItem.kt

```diff
@@ -50,7 +50,13 @@ object CommonMangaItemDefaults {
     const val BrowseFavoriteCoverAlpha = 0.34f
 }
 
-private val ContinueReadingButtonSize = 28.dp
+private val ContinueReadingButtonSize = mapOf(
+    "Compact" to 28.dp,
+    "Comfortable" to 35.dp,
+    "CoverOnly" to 35.dp,
+    "List" to 28.dp
+)
```

Comment:
make it 2 variables and use them instead of this jank

Reply:
also make it 32.dp

Reply:
Completed this now.